oak_cameras:
  "1944301081DB714800": #deviceId for oak0
    name: oak0
    params:
      camera:
        i_enable_imu: false
        i_enable_ir: false
        i_floodlight_brightness: 0
        i_laser_dot_brightness: 100
        i_nn_type: none
        i_pipeline_type: RGBD
        i_usb_speed: SUPER_PLUS
      rgb:
        i_board_socket_id: 0
        i_fps: 30.0
        i_height: 720
        i_interleaved: false
        i_max_q_size: 10
        i_preview_size: 250
        i_enable_preview: true
        i_low_bandwidth: true
        i_keep_preview_aspect_ratio: true
        i_publish_topic: false
        i_resolution: '1080P'
        i_width: 1280
      use_sim_time: false
  "19443010B1627E4800": #deviceId for oak1
    name: oak2
    params:
      camera:
        i_enable_imu: false
        i_enable_ir: false
        i_floodlight_brightness: 0
        i_laser_dot_brightness: 100
        i_nn_type: none
        i_pipeline_type: RGBD
        i_usb_speed: SUPER_PLUS
      rgb:
        i_board_socket_id: 0
        i_fps: 30.0
        i_height: 720
        i_interleaved: false
        i_max_q_size: 10
        i_preview_size: 250
        i_enable_preview: true
        i_low_bandwidth: true
        i_keep_preview_aspect_ratio: true
        i_publish_topic: false
        i_resolution: '1080P'
        i_width: 1280
      use_sim_time: false
usb_cameras:
  "2450ZB1354S8": # USB camera0 serial number
    name: camera0
    params:
      image_width: 1280
      image_height: 720
      pixel_format: mjpeg2rgb 
      auto_white_balance: false
      auto_exposure: false
      auto_focus: false
      framerate: 15.0